.changeset/plenty-eyes-mate.md

```diff
@@ -0,0 +1,7 @@
+---
+'@spectrum-web-components/contextual-help': patch
+---
+
+Fixed a typo in the default `info` variant label from "Informations" to "Information".
```

Comment:
Even though this was a tiny change, let me know if we should save it for a different PR. Because of this changing the main component file, eslint noted the missing package dependency as well.